azure-pipelines.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       condition: always()
     - script: dotnet msbuild -binaryLogger:artifacts\log\$(_configuration)\uploadCodeCov.binlog;ProjectImports=Embed -restore eng/CodeCoverage.proj
       displayName: Upload coverage to codecov.io
-      condition: and(succeeded(), eq(variables['_codeCoverage'], True))
+      condition: and(always(), eq(variables['_codeCoverage'], True)) # Upload coverage to codecov.io is flaky, so do not fail build on upload failure
```

Comment:
Is it possible to introduce some retry pattern?

Reply:
I don't see a reason for it - if someone really wants to look at codecov data and upload failed, they can manually re-run the debug queue on the PR. I don't believe queue should fail if upload of such additional information fails. We already take this approach if publishing test results fail, so this is identical.

Reply:
I agree with the fact it should not fail the build, I just think that the coverage metric is useful so having a retry seemed like a good option.